src/test/functional/utlis/puppeteer.util.ts

```diff
@@ -122,3 +122,20 @@ export const clearField = async (selector: string) => {
     console.log("The element didn't appear.");
   }
 };
+
+export const selectItem = async (selector: string, value: string) => {
+  try {
+    await scope.page.select(selector, value);
+  } catch (error) {
+    console.log("The element didn't appear.");
```

Comment:
Optional one again, you may want to add a distinct message to help debug, like "console.log("The element didn't appear for the selectItem function with selector {} and value {}.");

Reply:
Agreed. Code updated.